command/operator_migrate_test.go

```diff
@@ -55,11 +56,11 @@ func TestMigration(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-
+		maxParallel := "10"
```

Comment:
I'm not sure I follow how this tests the parallelism feature. Could you please elaborate on this?

Reply:
Hard to say after 2,5 years ;). Don't look like this makes any sens here. I set the parallel value to 10 only for DFS test and rest is set to 1.